src/NUnitFramework/FrameworkVersion.cs

```diff
@@ -26,5 +26,5 @@
 //
 // Current version for the NUnit Framework
 //
-[assembly: AssemblyVersion("3.10.0.0")]
-[assembly: AssemblyFileVersion("3.10.0.0")]
+[assembly: AssemblyVersion("3.11.0.0")]
+[assembly: AssemblyFileVersion("3.11.0.0")]
```

Comment:
Sometime can we start doing:

```cs
[assembly: AssemblyVersion("3.11.0")]
[assembly: AssemblyInformationalVersion("3.11.0")]
```

That will result in a .NET assembly version of 3.11.0.0 and Windows file version of 3.11.0.0 but a Windows product version of 3.11.0. (Can be seen in file properties.)

At work, we also have the build script update the file in CI before building to this effect:

```cs
[assembly: AssemblyInformationalVersion("3.11.0-ci-12345")]
```

Since the Windows product version is an arbitrary string, this works great. Some people also put `+{commit hash}` at the end.